tailwind.config.js

```diff
@@ -1,7 +1,133 @@
 /** @type {import('tailwindcss').Config} */
 const plugin = require("tailwindcss/plugin");
 const { createThemes } = require('tw-colors');
-import { createThemeFonts } from './src/utils/index';
+import { resolveTwcConfig } from './src/utils/index';
+
+const themeFontFamily = {
+	forest: {
+		body: '"Tenor Sans", sans-serif',
+    display: '"Merienda", cursive',
+  },
+  racer: {
+		body: "Montserrat",
+    display: "Anton",
+  },
+  "retro-diner": {
+		body: "Montserrat",
+    display: "Roboto Slab",
+  },
+};
+
+const tailwindColors = {
+	patchWRK: {
+		main: "white",
+		tertiary: "theme('colors.neutral.50')",
+		inactive: "theme('colors.blue.200')",
+		secondary: "theme('colors.slate.400')",
+		"accent-light": "theme('colors.slate.500')",
+		"trim-light": "theme('colors.slate.500')",
+		"accent-dark": "theme('colors.blue.700')",
+		primary: "theme('colors.blue.900')",
+		"trim-dark": "theme('colors.blue.900')",
+		info: "#0a0a0a",
+		success: "#22C55E",
+		error: "#F97316",
+		warning: "#DC2626",
+	},
+	nyias: {
+		main: "white",
+		tertiary: "theme('colors.neutral.50')",
+		inactive: "theme('colors.neutral.200')",
+		secondary: "theme('colors.red.400')",
+		"accent-light": "theme('colors.red.500')",
+		"trim-light": "theme('colors.red.500')",
+		"accent-dark": "theme('colors.neutral.700')",
+		primary: "theme('colors.neutral.900')",
+		"trim-dark": "theme('colors.neutral.900')",
+		info: "#0a0a0a",
+		success: "#22C55E",
+		error: "#F97316",
+		warning: "#DC2626",
+	},
+	dragons: {
+		main: "white",
+		tertiary: "theme('colors.neutral.50')",
+		inactive: "theme('colors.purple.200')",
+		secondary: "theme('colors.yellow.400')",
+		"accent-light": "theme('colors.yellow.500')",
+		"trim-light": "theme('colors.yellow.500')",
+		"accent-dark": "theme('colors.purple.700')",
+		primary: "theme('colors.purple.900')",
+		"trim-dark": "theme('colors.purple.900')",
+		info: "#0a0a0a",
+		success: "#22C55E",
+		error: "#F97316",
+		warning: "#DC2626",
+	},
+  grey: {
```

Comment:
do you use VSC for writing this stuff?
with Prettier extension?
so this looks more uniform

Reply:
@michael-gokey-architect Yes, and I have it set up to format on save, so I'm not sure how this part messed up.